beacon_node/store/src/lib.rs

```diff
@@ -175,8 +179,8 @@ pub fn parse_data_column_key(data: Vec<u8>) -> Result<(Hash256, ColumnIndex), Er
 #[must_use]
 #[derive(Clone)]
 pub enum KeyValueStoreOp {
-    PutKeyValue(Vec<u8>, Vec<u8>),
-    DeleteKey(Vec<u8>),
+    PutKeyValue(String, Vec<u8>, Vec<u8>),
+    DeleteKey(String, Vec<u8>),
```

Comment:
Document the tuple

Reply:
[ec7059b](https://github.com/sigp/lighthouse/pull/4718/commits/ec7059b172768ae2c5c92586b2db71bfbef316b5)